### PR TITLE
Create finops-assessment capability

### PR DESCRIPTION
**Pull request by ripadrao**
## 🛠️ Description

Fixes # 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Yes (required for `dev` PRs)
> - [ ] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### CODE REVIEW

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+<br>
+
+## 🏗️ Building on the basics
+
+As the organization gains more maturity in its capabilities, more capabilities will be assessed, based on FinOps KPIs. Each Business Unit leads its own FinOps Assessment at a regular cadence.
+
+## ⏩ Next steps
+
+<br>
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:35]: Should we include the "Learn more at the FinOps Foundation" section? 

## 🍎 Learn more at the FinOps Foundation

This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [FinOps Assessment capability](https://www.finops.org/framework/capabilities/finops-assessment/) article in the FinOps Framework documentation.

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+FinOps Capabilities can be evaluated at any point in an organization’s FinOps journey.
+The objective of the FinOps assessment is to capture the maturity of a set of activities, define the objectives according to the organization, and outline the path to achieve that goal. Setting KPIs will help the FinOps team identify which objectives are being met and which areas need more attention to increase maturity.
+When conducting an assessment, organizations can evaluate their current capabilities and reassess the capabilities that will have the most impact on maximizing cloud value. These assessments are carried out by a target group consisting of a group composed of different personas that bring different points of view on what is essential. As organizations gain maturity, they will implement more capabilities.
+
+<br>
+
+
+## ✋ Before you begin
+
+For organizations that are just starting out, they can conduct the Microsoft FinOps Review.
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:14]: Should we include a direct link to the FinOps Review assessment? 

[Microsoft FinOps Review](https://aka.ms/finops/review)

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+For organizations that are just starting out, they can conduct the Microsoft FinOps Review. 
+Leverage from FinOps guidance to assess your organization’s capability gaps. Start small, choose one or two capabilities to focus and that are a priority to maximize the cloud value. Finishing the assessment, you will get recommendations for maximizing cloud business value using FinOps best practices.
+Additionally, document all assessments to be available for the FinOps team and define annual goals with clear KPI’s based on the recommendations 
+
+
+<br>
+
+
+Getting started
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:22]: ## ▶️ Getting started

Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:22]:
```suggestion
## ▶️ Getting started
```

File: docs/_docs/framework/capabilities/manage/finops-assessment.md (hunk 2 above)
Comment (KevDLR) [reply docs/_docs/framework/capabilities/manage/finops-assessment.md:14]:
```suggestion
For organizations that are just starting out, they can conduct the [Microsoft FinOps Review](https://aka.ms/finops/review) 
```

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Find out what capabilities are relevant for the FinOps assessment, and confirm if KPIs are being met. The output of the FinOps assessment is used to monitor and evaluate annual objectives. If needed, adjust goals after each assessment.
+Additionally, build a consistent format that enables simpler comparisons between Target Groups and over time
+
+
+<br>
+
+## 🏗️ Building on the basics
+
+As the organization gains more maturity in its capabilities, more capabilities will be assessed, based on FinOps KPIs. Each Business Unit leads its own FinOps Assessment at a regular cadence.
+
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:32]:
```suggestion

🍎 Learn more at the FinOps Foundation
This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [FinOps Assessment capability](https://www.finops.org/framework/capabilities/finops-assessment/) article in the FinOps Framework documentation.
```

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+The objective of the FinOps assessment is to capture the maturity of a set of activities, define the objectives according to the organization, and outline the path to achieve that goal. Setting KPIs will help the FinOps team identify which objectives are being met and which areas need more attention to increase maturity.
+When conducting an assessment, organizations can evaluate their current capabilities and reassess the capabilities that will have the most impact on maximizing cloud value. These assessments are carried out by a target group consisting of a group composed of different personas that bring different points of view on what is essential. As organizations gain maturity, they will implement more capabilities.
+
+<br>
+
+
+## ✋ Before you begin
+
+For organizations that are just starting out, they can conduct the Microsoft FinOps Review. 
+Leverage from FinOps guidance to assess your organization’s capability gaps. Start small, choose one or two capabilities to focus and that are a priority to maximize the cloud value. Finishing the assessment, you will get recommendations for maximizing cloud business value using FinOps best practices.
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:15]:
```suggestion
Leverage the FinOps guidance to assess your organization’s capability gaps. Start small, choose one or two capabilities to focus on that are a priority to maximize the cloud value. Finishing the assessment, you will get recommendations for maximizing cloud business value using FinOps best practices.
```

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+Additionally, document all assessments to be available for the FinOps team and define annual goals with clear KPI’s based on the recommendations
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:16]:
```suggestion
Additionally, document all assessments to be available for the FinOps team and define annual goals with clear KPI’s based on the recommendations.
```

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+<a name="definition"></a>
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:1]:
```suggestion
<details open markdown="1">
  <summary class="fs-2 text-uppercase">On this page</summary>

- [✋ Before you begin](#-before-you-begin)
- [▶️ Getting started](#️-getting-started)
- [🏗️ Building on the basics](#️-building-on-the-basics)
- [🍎 Learn more at the FinOps Foundation](#-learn-more-at-the-finops-foundation)
- [⏩ Next steps](#-next-steps)

</details>

---

<a name="definition"></a>
```

File: docs/_docs/framework/capabilities/manage/finops-assessment.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+<br>
+
+## 🏗️ Building on the basics
+
+As the organization gains more maturity in its capabilities, more capabilities will be assessed, based on FinOps KPIs. Each Business Unit leads its own FinOps Assessment at a regular cadence.
+
+## 🍎 Learn more at the FinOps Foundation
+This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [FinOps Assessment capability](https://www.finops.org/framework/capabilities/finops-assessment/) article in the FinOps Framework documentation.
+## ⏩ Next steps
+
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/manage/finops-assessment.md:55]:
```suggestion

Related FinOps capabilities:

- [Benchmarking](../optimize/benchmarking.md)
- [Reporting and analytics](../understand/reporting.md)

```

**Review by flanakin — APPROVED** [review]

**Pull request merged**